requirements.txt

```diff
@@ -1,4 +1,4 @@
 django>=1.11,<3.0
 djangorestframework<4.0
-social-auth-core<2.0
+social-auth-core<=3.0
```

Comment:
Please set it to `<4.0`.
Reason: all releases of `social-auth-core` with version 3.x.x should be acceptable since they should be backward compatible. We don't want to update the requirements of this lib when `3.0.1` release will be available.

Reply:
Updated in the latest commit.